tests/client_test.py

```diff
@@ -254,5 +258,60 @@ def setUp(self):
         self.uut = SwaggerClient.from_resource_listing(self.resource_listing)
 
 
-if __name__ == '__main__':
-    unittest.main()
+class MockHttpClient(object):
+
+    def __init__(self, server_response):
+        self.response = server_response
+
+    def request(self, request, response_callback):
+        return HttpFuture(
+            mock.Mock(),
```

Comment:
Please make the type of the object that the Mock is mocking obvious using `spec`.